### PR TITLE
Azure Managed Kubernetes Cluster

### PR DESCRIPTION
**Pull request by pupapaik**
AKS provisioner for managed kubernetes cluster.

### CODE REVIEW

File: kqueen/engines/test_aks.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from .aks import AksEngine
+
+
+class TestAksInit:
+    def setup(self):
```
Comment (tomkukral) [inline kqueen/engines/test_aks.py]: This setup can be deleted.

File: kqueen/engines/test_aks.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        pass
+
+    def test_init_aks(self, cluster, monkeypatch):
+        def fake_client(self):
+            return True
+
+        monkeypatch.setattr(AksEngine, "_get_client", fake_client)
```
Comment (tomkukral) [inline kqueen/engines/test_aks.py]: This isn't a blockt but we are trying to uses consistent quotes - `'`

File: run_aks_test.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
```
Comment (tomkukral) [inline run_aks_test.py]: Is this file required to be merged into master?

File: setup.py [hunk 4]
```diff
@@ -44,6 +44,8 @@
         'google-api-python-client==1.6.4',
         'google-auth==1.2.1',
         'google-auth-httplib2==0.0.3',
+        'azure',
```
Comment (tomkukral) [inline setup.py]: Maybe we should pin versions?

File: run_aks_test.py (hunk 3 above)
Comment (pupapaik) [reply run_aks_test.py]: it is example for aks testing
